web/frontend/package.json

```diff
@@ -14,12 +14,10 @@
   },
   "dependencies": {
     "@dedis/kyber": "^3.4.4",
-    "@emotion/react": "^11.7.1",
-    "@emotion/styled": "^11.6.0",
     "@headlessui/react": "^1.5.0",
     "@heroicons/react": "^1.0.6",
-    "@material-ui/core": "^4.12.3",
-    "@mui/material": "^5.4.0",
+    "@material-ui/core": "^4.12.4",
```

Comment:
I don't see any imports inside the code of material UI , I think we can remove it .

Reply:
Great, nice catch !